cScripts/CavFnc/functions/systems/fn_vehicle_setRadio.sqf

```diff
@@ -0,0 +1,71 @@
+#include "..\script_component.hpp";
+/*
+ * Author: CPL.Brostrom.A
+ * This function changes the inventory of the given vehicle.
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ *
+ * Example:
+ * ["vic"] call cScripts_fnc_vehicle_setRadio;
+ */
+#define DEBUG_MODE
+
+params [["_vehicle", objNull, [objNull]]];
+
+if (didJIP) exitWith {};
+if (!isServer) exitWith {};
+if (_vehicle iskindOf "man") exitWith {};
+if (!isNil {_vehicle getVariable QEGVAR(Vehicle,Radio);}) exitWith {[formatText["Vehicle Radio already applied for %1.", _vehicle], "Vehicle"] call FUNC(logWarning);};
+if (!(faction _vehicle in ["BLU_F", "BLU_CTRG_F", "BLU_T_F", "BLU_USA_7CAV_F", "rhs_faction_usaf", "rhs_faction_usarmy", "rhs_faction_usarmy_d", "rhs_faction_usarmy_wd", "rhs_faction_usmc", "rhs_faction_usmc_d", "rhs_faction_usmc_wd", "rhs_faction_usn"]));
```

Comment:
```suggestion
```suggestion
private _factionArray = parseSimpleArray EGVAR(Settings,vehicleFactions);
if !(_factionArray isEqualType []) exitWith {["Faction array have not been setup correctly.", "Vehicle"] call FUNC(error);};
if (!(faction _vehicle in _factionArray));
```
```